docker/publication-server-docker.conf

```diff
@@ -0,0 +1,56 @@
+#
+# Docker specific application settings and environment variables as overrides.
+# ${?ENV_VAR} syntax overrides with ENV_VAR if the environment variable exists.
+#
+
+publication {
+  port = 7766
+  # if you use HTTPS for publication server (publication.spray.can.server.ssl-encryption == on)
```

Comment:
Just wondering if this variable from spray is still used. We migrated from spray to akka-http, maybe akka-http still inherit this property variable?